services/gcp/cloudkms/cryptoKeyVersions.yml

```
@@ -0,0 +1,77 @@
+name: Cloud KMS Crypto Key Versions
+description: >-
+  A key version contains key material used for encryption or signing.
+scope: CRITICAL
+notes: >-
+  Cloud KMS is an extremely sensitive service. Keys can be used for encryption/decryption of 
+  sensitive data or for the creation or verification of digital signatures.
+privileges:
+  create:
+    risks: [impact:spend]
+  destroy:
+    risks: [destruction:crypto, impact:encryption]
+    notes: >-
+      Destroyed key versions cannot be recovered.
+      Any data encrypted with the key version will no longer be able to be decrypted.
+  get:
+    risks: [discovery:infra]
+  list:
+    risks: [discovery:infra]
+  manageRawAesCbcKeys:
+    risks: []
+    notes: >-
+      This permission is required to manage AES CBC keys. 
+      It has no risks because it provides no access on its own: the user
+      still needs permissions on the keys.
+  manageRawAesCtrKeys:
+    risks: []
+    notes: >-
+      This permission is required to manage AES CTR keys. 
+      It has no risks because it provides no access on its own: the user
+      still needs permissions on the keys.
+  manageRawPKCS1Keys:
+    risks: []
+    notes: >-
+      This permission is required to manage AES CTR keys. 
+      It has no risks because it provides no access on its own: the user
+      still needs permissions on the keys.
+  restore:
+    risks: [impact:spend]
+    notes: >-
+      Restores a key that was scheduled for destruction.
+  update:
+    risks: [impact:dos, destruction:metadata]
```

What about `destruction:crypto`?
```suggestion
    risks: [impact:dos, destruction:metadata, destruction:crypto]
```

---

Updating a key version does not actually let you destroy the key or edit the secret contents in any way, so it wouldn't be destcrution:crypto.